### PR TITLE
add force flag for update

### PR DESCRIPTION
**Pull request by sascha-andres**
This PR suppresses the prompt asking to proceed with update, instead it goes straight to the update process

### CODE REVIEW

**Review by marcosnils — COMMENTED** [review]
Thx for the contribution! 

As a general comment, I don't think `--force` is the correct flag for this process. I think something like what `apt` does with `--yes / -y` is more convenient. Since `force` can be used for something different here.

File: cmd/update.go [hunk 1]
```diff
@@ -84,21 +86,23 @@ func newUpdateCmd() *updateCmd {
...
+				// so it can be reused in some other places
+				fmt.Printf("\nDo you want to continue? [Y/n] ")
+				reader := bufio.NewReader(os.Stdin)
+				var response string
 
-			response, err := reader.ReadString('\n')
-			if err != nil {
-				return fmt.Errorf("Invalid input")
-			}
+				response, err := reader.ReadString('\n')
```
[the remark this answers is not in the document]
Comment (sascha-andres) [reply cmd/update.go:94]: I thought I have, but my config was not complete after a re-install of my machine

**Comment by sascha-andres** [comment]
@marcosnils Any further comments?

File: cmd/update.go [hunk 2]
```diff
@@ -139,6 +141,7 @@ func newUpdateCmd() *updateCmd {
 
 	root.cmd = cmd
 	root.cmd.Flags().BoolVarP(&root.opts.dryRun, "dry-run", "", false, "Only show status, don't prompt for update")
+	root.cmd.Flags().BoolVarP(&root.opts.yesToUpdate, "yes", "y", false, "Allow updates (do not ask)")
```
Comment (sirlatrom) [inline cmd/update.go:144]: How about this wording?

```suggestion
	root.cmd.Flags().BoolVarP(&root.opts.yesToUpdate, "yes", "y", false, "Assume yes to update prompt")
```

**Comment by sascha-andres** [comment]
@sirlatrom Agreed, is a better wording

**Comment by marcosnils** [comment]
There still seem to be some formatting issues with the code. Did you run `go fmt`?

**Comment by sascha-andres** [comment]
If I run go fmt nothing changes locally, so it seems the run was successful. My IDE is configured to run it on save.

**Comment by marcosnils** [comment]
> If I run go fmt nothing changes locally, so it seems the run was successful. My IDE is configured to run it on save.

That's correct. Not sure why github diff is showing those green marks though :confused: . I thought it was because it's incorrectly indented but again, doesn't seem to be the case.

**Review by marcosnils — APPROVED** [review]

**Comment by sirlatrom** [comment]
> 
> 
> > If I run go fmt nothing changes locally, so it seems the run was successful. My IDE is configured to run it on save.
> 
> That's correct. Not sure why github diff is showing those green marks though 😕 . I thought it was because it's incorrectly indented but again, doesn't seem to be the case.
> 
> @sirlatrom WDYT?

Sorry for not getting back to you earlier. LGTM!

**Comment by cristiand391** [comment]
Thank you @sascha-andres for working on this!
